shared/actions/authActions.js

```diff
@@ -3,6 +3,7 @@ import { AsyncStorage } from 'react-native';
 
 const STORAGE_KEY_accessToken = '@PingoStorage:accessToken';
```

Comment:
'@PingoStorage:' is duplication
I think that it is good to define a variable and put '@PingoStorage:'.
Like this,
const STORAGE_PRE = '@PingoStorage:';
const STORAGE_KEY_refreshToken = STORAGE_PRE + 'refreshToken';

How about you guys?